docs/download_files.md

```diff
@@ -7,8 +7,10 @@ The folder structure within the zip file is determined by whether individual sam
 ## Download folder structure for project downloads:
 ![docs-downloads-project](https://user-images.githubusercontent.com/15315514/143308420-a3cca10d-814f-4c52-b934-98d5e9cef1c5.png)
 
-If a project contains bulk RNA-seq samples, a single tab-separated values file, `SCPCP000000_bulk_quant.tsv` will be included in the download. 
-This file contains a gene by sample matrix (each row a gene, each column a sample) containing raw gene expression counts quantified by Salmon.
+If a project contains bulk RNA-seq samples, two tab-separated value files, `SCPCP000000_bulk_quant.tsv` and `SCPCP000000_bulk_metadata.tsv`, will be included in the download. 
+The `SCPCP000000_bulk_quant.tsv` file contains a gene by sample matrix (each row a gene, each column a sample) containing raw gene expression counts quantified by Salmon.
+The `SCPCP000000_bulk_metadata.tsv` file contains associated metadata, such as the type of RNA-seq, total number of reads, and processing information for each sample.  
```

Comment:
Are the fields largely the same as for the single cell data? Can we combine the descriptions for those two tables into a single section? 

More generally, are we copying the sample metadata into this table at import as well, or is this staying separate? That is a question for @davidsmejia I suppose. Maybe also for @dvenprasad, as to what we _should_ do. 

For more context, the bulk sample metadata we provide for import has library information related to the assay, but does not contain sample information such as disease type. We _do_ merge in that information in the `library_metadata.tsv` file, as described below in the **Metadata** section, but that file only includes the single cell libraries.

I think it makes sense to keep the two data tables separate, despite the fact that many fields will be similar if we do merge in metadata, but I wonder if we might want to rename `library_metadata.tsv` below to match the naming of these files, i.e., `SCPCP000000_sc_metadata.tsv`.

Reply:
> Are the fields largely the same as for the single cell data? Can we combine the descriptions for those two tables into a single section?

In looking at a test download of the single-cell data and the fields provided in `libraries_metadata.csv` in compared to what we provide in the `bulk_metadata.tsv`, the `bulk_metadata.tsv` columns are mostly all found in the single-cell metadata. The single-cell metadata has some additional single-cell specific columns like filtered cell counts and then the additional metadata for that sample like disease, age, etc. 
I do want to note that there are some additional columns that I noticed that are not currently accounted for in the docs, `salmon_version`, `alevin_fry_version`, `unfiltered_cells`, `total_reads`, `mapped_reads`, `transcript_type`, and `cell_count` (? this also appears to duplicate filtered cell count). It might be worth adding some of these to our table in the docs. 

I think we could add a sentence in the metadata section stating that the bulk file only contains processing information and does not contain columns related to sample metadata. Unless we think it would be important to have it repeated in both places? 

> I think it makes sense to keep the two data tables separate, despite the fact that many fields will be similar if we do merge in metadata, but I wonder if we might want to rename library_metadata.tsv below to match the naming of these files, i.e., SCPCP000000_sc_metadata.tsv.

I do like the idea of renaming the single-cell metadata to match the bulk metadata.

Reply:
> More generally, are we copying the sample metadata into this table at import as well, or is this staying separate? That is a question for @davidsmejia I suppose. Maybe also for @dvenprasad, as to what we should do.

Will be separate, nested inside of the project folder.

I renamed the files to: 
`libraries_metadata.csv` => `single_cell_metadata.csv` 
`SCPCP000000_bulk_metadata.tsv` =>  `bulk_metadata.tsv`
`SCPCP000000_bulk_quant.tsv` => `bulk_quant.tsv`

They will be nested in the project folder so I think we should either consistently prefix with project id or omit it.

Reply:
> I renamed the files to:
libraries_metadata.csv => single_cell_metadata.csv
SCPCP000000_bulk_metadata.tsv => bulk_metadata.tsv
SCPCP000000_bulk_quant.tsv => bulk_quant.tsv

I like this change. If @jashapiro is onboard then I'll update the docs to reflect these changes but we will also need to update the screenshots.

Reply:
I like it, but for one thing... `single_cell_metadata.csv` will come after `SCPCL` alphabetically, so those files won't sort to the top(or together). But I don't think that is worth modifying the name for.

Reply:
I don't think it matters theoretically, but it's probably a good idea to convert them all to the same format for consistency. Changing the bulk metadata to `csv` would require updates to the workflow and re-running the projects since that is generated through `scpca-nf`, which shouldn't be a huge hassle but is not minimal effort. I'm assuming that changing the `libraries_metadata.csv` to `tsv` would also require changes to creating that file on your end though @davidsmejia so the question is which one is the least disruptive and how important is it to change it.

Reply:
I can change `libraries_metadata.csv` -> `single_cell_metadata.csv` -> `single_cell_metadata.tsv`

Reply:
My brain clearly was not working, I forgot both changes would be pretty easy. But that works 👍